src/client/components/Divider.tsx

```diff
@@ -36,11 +38,50 @@ const decideSpace = (
     case 'loose':
       return displayText
         ? css`
-            margin-top: 37px;
+            margin-top: ${space[6]}px;
           `
         : css`
             margin-top: ${space[12]}px;
           `;
+    case 'none':
+      return css`
+        margin-top: 0;
+      `;
+  }
+};
+
+const decideSize = (size: Sizes) => {
+  switch (size) {
+    case 'fit':
+      return css`
```

Comment:
Added the `fit` size for cases such as the registration page where the divider shouldn't overlap the container

Reply:
To be added to Source Kitchen instead